src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/JDepsGenerator.kt

```diff
@@ -53,6 +53,7 @@ internal class JDepsGenerator @Inject constructor(
                         val joinedClasspath = command.inputs.joinedClasspath
                         val res = invoker.run(
                             arrayOf(
+                                "--multi-release", "base",
```

Comment:
Setting "base" means we use `java.util.jar.JarFile.baseVersion()` - is that a good value?

Do we need to guard this in case the flag is not present on older versions?

Reply:
Yes, added a guard.

Reply:
Yeah, we definitely need the guard.